README.md

```diff
@@ -1,2 +1,10 @@
 # number-partitioning
 Number partitioning problem coding exercise.
+
+The number partitioning problem begins with a set of numbers, S.  We must split the set of numbers into two sets with equal sum.  In our training course, we discuss this problem and how to create a QUBO to solve it.
+
+In this exercise, students are given an Ocean program outline and must create the QUBO dictionary (Q) that is provided to the D-Wave QPU and choose appropriate values for `chainstrength` and `numruns` on lines 24-25.
```

Comment:
Perhaps don't specify the lines 24-25? `chainstrength` and `numruns` are only defined in one place, so the students should know where to find it. As well, if anything changes in the code (like adding in a QUBO to the code), the readme text is still accurate.

Reply:
fixed